### ISSUE TITLE
Error while training with pretrained vector 

### ISSUE THREAD

**Issue by Wisharyco** [facebookresearch/fastText#820]
I've downloaded the Korean pre-trained vectors from the fasttext website.  However, when I run it with the command:
./fasttext supervised -input /home/wishary/nlp/all_text \
       -output /home/wishary/nlp/fasttext_vocab/retrain_kr \
       -dim 300 -loss hs \
       -pretrainedVectors /home/wishary/nlp/fasttext_vocab/ko.300.vec, 
I get the following error:
terminate called after throwing an instance of 'std::length_error'
   what():  vector::_M_default_append
 Aborted (core dumped)

when I train it from the scratch, everything seems working.
